src/components/GobusterTool/Gobuster.tsx

```diff
@@ -52,8 +104,8 @@ const GoBusterTool = () => {
                 {UserGuide(title, description_userguide)}
                 <TextInput label={"Target URL"} required {...form.getInputProps("url")} />
                 <TextInput label={"Wordlist File"} required {...form.getInputProps("wordlist")} />
-                <TextInput label={"Output File (optional)"} {...form.getInputProps("outputFile")} />
                 <Button type={"submit"}>Start Brute-Forcing</Button>
+                {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
```

Comment:
This is not a bug. `-o` is a feature that allow the user to save the output result to a file name. However, with that being said, it is very similar to `{SaveOutputTotextFile_v2}`. The difference is `{SaveOutputTotextFile_v2}` save the whole command include all the print value during the execution but `-o` only print the result.